src/enums/regions.ts

```diff
@@ -1,4 +1,5 @@
 export const regionMap = {
+  global: 'global',
```

Comment:
I would keep this out of the region map as we dont want it to show up when running `cg init tencent`

Reply:
@tyler-dunkel I found we have the GLOBAL_REGION constant already. Updated to reuse the constant.